app/shared/components/design-system/all-components/checkbox/Checkbox.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+import CustomCheckbox from './Checkbox';
+import userEvent from '@testing-library/user-event';
+
+describe('CustomCheckbox', () => {
+  const label = 'Label';
+  it('should display checkbox', async () => {
```

Comment:
```suggestion
  it('should display checkbox', () => {
```
Please, remove this unnecessary async